core-bundle/src/Resources/config/services.yml

```
@@ -110,6 +116,19 @@ services:
         tags:
             - { name: nelmio_cors.options_provider }
 
+    Contao\CoreBundle\Cron\Cron:
+        arguments:
+            - '@database_connection'
+            - '@logger'
```

IIRC the logger should be optional

```suggestion
            - '@?logger'
```